atom/browser/api/atom_api_power_monitor.cc

```diff
@@ -38,8 +39,14 @@ void PowerMonitor::OnResume() {
 }
 
 // static
-mate::Handle<PowerMonitor> PowerMonitor::Create(v8::Isolate* isolate) {
-  return CreateHandle(isolate, new PowerMonitor);
+v8::Handle<v8::Value> PowerMonitor::Create(v8::Isolate* isolate) {
+  if (!Browser::Get()->is_ready()) {
+    node::ThrowError(R"(Cannot initialize "power-monitor" module
```

Comment:
We should not use raw string literals for now because it causes troubles on some compilers, see https://chromium-cpp.appspot.com for more.